modules/sonarqube-service/main.tf

```diff
@@ -26,12 +26,13 @@ module "sonarqube-service" {
     load_balancer = "${var.loadbalancer_arn}"
   }
 
-  task_container_image              = "teliaoss/sonarqube-aws-env:7.3"
+  task_container_image              = "teliaoss/sonarqube-aws-env:7.3b"
```

Comment:
Is that tag just created for testing purposes? If so I think you should publish a new `7.3`... however that might break existing deployments 🤔 Perhaps it is best to just version the docker image along with the terraform module, since one does not work without the other?

Reply:
I have been versioning the docker images with the version of sonarqube and adding a letter to the end of the tags if I need to create a new release of the wrapper so as not to change existing deployments.  It probably wouldn't break things this time if I reused the existing tag but if I follow this pattern for all similar deploys then I can avoid breaking existing deployments